### DISCUSSION TITLE
Griswold shopping question

### DISCUSSION THREAD

**Discussion by smloeffelholz** [diasurgical/DevilutionX#6366] [Q&A] [answered]
I am using the DevilutionX mod ver 1.5.0 to play Diablo with Hellfire features enabled. I just started trying to shop at Griswold for some better items. I am doing this with my level 34 sorceror. I have never seen full plate mail, a ring, or an amulet as a base item despite opening his shop around 100 times. Is this just bad luck, or are these bases not available in multiplayer/Hellfire?

**Comment by DakkJaniels** [comment, marked as answer]
Rings and amulets are not available from Griswold in multiplayer.
Griswold will try and offer you armor that costs more than 80% of the value of the armor you currently are wearing or have in your inventory (not stash). Wear more expensive armor and you might see full plate mail as a base item.
